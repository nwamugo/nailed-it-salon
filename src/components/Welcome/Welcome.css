.Welcome {
  position: absolute;
  width: 100%;
  height: 50.625rem;
  top: 5.8125rem;
  background: url(welcome.jpg);
  background-size: cover;
}
.welcome-tag {
  position: relative;
  width: 25.75rem;
  height: 6.25rem;
  left: 3.75rem;
  top: 20.5625rem;
  font-style: normal;
  font-weight: 600;
  font-size: 13.25rem;
  line-height: 6.25rem;
  letter-spacing: 0.01em;
  color: #000000;
}
.sub-tag {
  position: relative;
  width: 28.0625rem;
  height: 1.25rem;
  left: 4.375rem;
  top: 22.925rem;

  font-family: 'Libre Baskerville', Roboto, serif;
  font-style: normal;
  font-weight: normal;
  font-size: 2.2rem;
  line-height: 1.25rem;
  letter-spacing: 0.01em;

  color: #000000;
}
