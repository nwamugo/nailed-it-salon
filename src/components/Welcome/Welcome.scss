.Welcome {
  position: absolute;
  width: 100%;
  height: 81rem;
  top: 9.3rem;
  background: url(welcome.jpg);
  background-size: cover;

  @media (max-width: 26.75em) { 
    background-image: url(welcome500.jpg);
    background-position: 64% 6%;
  }
}
.welcome-tag {
  position: relative;
  width: 41.2rem;
  height: 10rem;
  left: 6rem;
  top: 32.9rem;
  font-style: normal;
  font-weight: 600;
  font-size: 21.2rem;
  line-height: 10rem;
  letter-spacing: 0.01em;
  color: #000000;
}
.sub-tag {
  position: relative;
  width: 44.9rem;
  height: 2rem;
  left: 7rem;
  top: 36.68rem;

  font-family: 'Libre Baskerville', Roboto, serif;
  font-style: normal;
  font-weight: normal;
  font-size: 3.52rem;
  line-height: 2rem;
  letter-spacing: 0.01em;

  color: #000000;
}
