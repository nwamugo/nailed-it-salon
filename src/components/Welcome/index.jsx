/* eslint-disable no-irregular-whitespace */
import React from 'react';
import './Welcome.scss';

const Welcome = () => {
  return (
    <div className="Welcome">
      <span className="welcome-tag">Welcome</span>
      <br />
      <span className="sub-tag">To a Full Nail Salon Experience</span>
    </div>
  );
};

export default Welcome;
