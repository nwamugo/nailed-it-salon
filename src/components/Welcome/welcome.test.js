import React from 'react';
import { shallow } from 'enzyme';
import Welcome from './index';

describe('Welcome', () => {
  it('should render correctly', () => {
    const component = shallow(<Welcome />);

    expect(component).toMatchSnapshot();
  });
});
