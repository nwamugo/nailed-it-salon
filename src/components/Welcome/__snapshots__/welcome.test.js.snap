// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<div
  className="Welcome"
>
  <span
    className="welcome-tag"
  >
    Welcome
  </span>
  <br />
  <span
    className="sub-tag"
  >
    To a Full Nail Salon Experience
  </span>
</div>
`;
